tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -223,14 +228,13 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
       assert.include hls, anchor2.highlight[anchor2.startPage]
       assert.notInclude hls, anchor3.highlight[anchor3.startPage]
 
-  describe 'Anchor.scrollIntoView()', ->
+  describe 'Anchor.scrollToView()', ->
     it 'calls scrollIntoView() on the highlight', ->
```

Comment:
> I see this is tested below. Not sure if you should make explicit here that we're _not_ using two phases.

Well, I suppose I could add a `describe "single-phase anchoring"` block around the single-phase tests...